framework/test/fixtures/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": ["../../.eslintrc.json"],
+	"rules": {
+		"import/no-extraneous-dependencies": [
+			"error",
+			{
+				"devDependencies": ["./**"]
```

Comment:
why do we need to add this rule?

Reply:
Copied it over from existing structure. Its overrule the warning of putting some dependencies in the `dependencies` rather `devDependencies`

Reply:
it seems it will not throw lint error without this